src/main/java/edu/uz/dr/auctioneer/model/request/AuctionRequest.java

```diff
@@ -101,13 +98,17 @@ public void setUserDescription(final String userDescription) {
         this.userDescription = userDescription;
     }
 
+    public static String getDateFormat() {
```

Comment:
pojo tester ma tak ze albo testujemy wszystkie gettery lub settery z klasy albo nic. A przy odpalaniu wyskakuje wyjatek "has no setter for field private static final". Nie da sie jednego pominac